src/main/java/org/assertj/core/error/ElementsShouldBe.java

```diff
@@ -35,7 +35,7 @@ public static ErrorMessageFactory elementsShouldBe(Object actual, Object notSati
   }
 
   private ElementsShouldBe(Object actual, Object notSatisfies, Condition<?> condition) {
-    super("%nExpecting elements:%n<%s>%n of %n<%s>%n to be <%s>", notSatisfies, actual, condition);
+    super("%nExpecting elements:%n%s%n of %n%s%n to be %s", notSatisfies, actual, condition);
```

Comment:
Fix the indentation to be like `"%nExpecting elements:%n  %s%nof%n  %s%nto be %s"`

Reply:
apply the same indentation logic to other ElementsShouldBeXXX classes

Reply:
I am adding them for all ElementsShouldBeXXX classes.

Reply:
@joel-costigliola  Covered all ElementsShouldBeXXX classes with suggested indentation. Do you want a similar indentation for all ElementsShouldHaveXXX classes as well?